roles/bats/defaults/main.yaml

```diff
@@ -20,4 +20,3 @@ bats_tests:
 bats_tests_additional: []
 bats_teardown:
   - "fb-destroy-organization.bats"
```

Comment:
I now wonder if it should be empty by default and opt-in, but that doesn't have to block this.